test/sumo_test_people_riak.erl

```diff
@@ -1,22 +1,34 @@
 -module(sumo_test_people_riak).
 
--behavior(sumo_doc).
-
--include_lib("mixer/include/mixer.hrl").
--mixin([{sumo_test_people,
-         [sumo_wakeup/1,
-          sumo_sleep/1,
-          new/2,
-          new/3,
-          new/4,
-          name/1,
-          id/1,
-          age/1
-         ]
-        }
-       ]).
-
--export([sumo_schema/0]).
+%%% sumo_db callbacks
+-export([ sumo_schema/0
+        , sumo_wakeup/1
+        , sumo_sleep/1
+        ]).
+
+-export([ new/2
+        , new/3
+        , new/4
+        , name/1
+        , id/1
+        , age/1
+        , date/1
+        , datetime/1
+        ]).
+
+-record(person, {id        :: integer(),
+                 name      :: string(),
+                 last_name :: string(),
+                 age       :: integer(),
+                 address   :: string(),
+                 date      :: calendar:date(),
```

Comment:
Try not to call the fields in such a generic way (even for tests). Call them `birthdate` and `last_login` or `created_at`, if you will.